### DISCUSSION TITLE
How to use /dev branch?

### DISCUSSION THREAD

**Discussion by tomByrer** [nuejs/nue#380] [Q&A] [answered]
I've noticed quite a bit of action in [/dev](https://github.com/nuejs/nue/tree/dev); how do I install & use that branch instead of the release?

Since nue is beta anyhow, I'd rather use the latest beta than an old beta :)

**Comment by nobkd** [comment, marked as answer]
In short, there's no dev/edge release, so you'll have to install the dev version using git:

1. Remove installed nuekit version from your system `bun rm -g nuekit`
2. clone the repo `git clone git@github.com:nuejs/nue`
3. `cd nue`
4. switch to dev branch `git switch dev`
5. `bun install` on project root
6. `bun link` from `packages/nuekit`

In one line: `bun rm -g nuekit; git clone git@github.com:nuejs/nue; cd nue; bun i; cd packages/nuekit; bun link`

now you can use `nue` CLI globally.

I've written this down from memory, so maybe I missed a step. Good luck.

---

If you want to remove the dev version, use `bun unlink` in `packages/nuekit`

---

PS: if you are on windows, I think you'll have to rerun the `link` step each time, you pull from remote / change the code. 

PPS: expect bugs on the dev branch
